### PR TITLE
fix some formatting issues in STYLE-GUIDE.md

### PR DESCRIPTION
**Pull request by rindeal**

### CODE REVIEW

File: STYLE-GUIDE.md [hunk 1]
```diff
@@ -51,8 +51,8 @@ the body entirely); tools like rebase can get confused if you run the
 two together.
 
 Write your commit message in the imperative: "Fix bug" and not "Fixed
-bug."  This convention matches up with commit messages generated by
-commands like git merge and git revert.
+bug". This convention matches up with commit messages generated by
```
Comment (dwfreed) [inline STYLE-GUIDE.md]: Periods and commas are always placed inside the quotations marks. (http://blog.apastyle.org/apastyle/2011/08/punctuating-around-quotation-marks.html)  However, do not apply this to the next line, as those aren't considered quotes.

File: STYLE-GUIDE.md [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
+This is the openrc style manual. It governs the coding style of all code
+in this repository. Follow it. Contact openrc@gentoo.org for any questions
 or fixes you might notice.
 
 ## C CODE
 
-The BSD Kernel Normal Form (KNF) style is used [1]. Basically, it is like
-K&R/LKML, but wrapped lines that are indented use 4 spaces. Here are the
+The BSD Kernel Normal Form (KNF) style is used [[1]]. Basically, it is like
+K&R/LKML, but wrapped lines, that are indented, use 4 spaces. Here are the
```
Comment (dwfreed) [inline STYLE-GUIDE.md]: This is not a parenthetical; do not put commas here

File: STYLE-GUIDE.md [hunk 3]
```diff
@@ -1,19 +1,19 @@
 # OpenRC Style Guide
 
-This is the openrc style manual.  It governs the coding style of all code
-in this repository.  Follow it.  Contact openrc@gentoo.org for any questions
+This is the openrc style manual. It governs the coding style of all code
```
Comment (dwfreed) [inline STYLE-GUIDE.md]: Let's not starts the 1 space versus 2 spaces war.  Leave it at 2 spaces.

File: STYLE-GUIDE.md [hunk 4]
```diff
@@ -38,21 +38,21 @@ void foo(int c)
 
 The following is an example of a correctly formatted git commit message
 for this repository. Most of this information came from this blog post
-[2], so I would like to thank the author.
+[[2]], so I would like to thank the author Tim Pope.
```
Comment (dwfreed) [inline STYLE-GUIDE.md]: There should be a comma after author.

File: STYLE-GUIDE.md [hunk 5]
```diff
@@ -38,21 +38,21 @@ void foo(int c)
...
 
 ```
 Capitalized, short (50 chars or less) summary
 
-More detailed explanatory text, if necessary.  Wrap it to about 72
+More detailed explanatory text, if necessary. Wrap it to about 72
```
Comment (dwfreed) [inline STYLE-GUIDE.md]: Space war

File: STYLE-GUIDE.md (hunk 3 above)
Comment (rindeal) [inline STYLE-GUIDE.md]: I didn't know there is another styling except for a single space. I have always considered two spaces to be typos.

**Comment by rindeal** [comment]
Thank you for your comments, but I don't intend to maintain this patch any more. So feel free to cherry-pick it, implement your suggestions and send a new PR.

**Pull request closed without merging**